src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -64,92 +50,57 @@ export default function EpisodePreview(props){
 		});
 	}// setOverlay
 
-
 	// BINDING
 	// ------------------------------
 	const prevEpisode  = setActiveIndex.bind(this, index - 1);
 	const nextEpisode  = setActiveIndex.bind(this, index + 1);
-	
-	const openOverlay  = setOverlayActive.bind(this, true);
-	const closeOverlay = setOverlayActive.bind(this, false);
 
-	
-	// RENDER LOGIC
-	// ----------------------------
-	// handle visibility of active episode and episode preview overlay
-	const isActiveEpisode   = activeIndex === index;
-	const isOverlayActive   = isActiveEpisode && overlayActive;
-	const hidden            = !isOverlayActive;
-
-	const isFirst           = index === 0;
-	const isLast            = index === episodeCount - 1;
+	const closeOverlay = setOverlayActive.bind(this, false);
+	const openOverlay  = setOverlayActive.bind(this, true);
 
+	// RENDER
+	// ---------------------------
 	const formattedTitle    = UTILS.convertToSafeString(title, "-");
 	const previewId         = `${formattedTitle}-preview`;
-
-	const { src: prevTrailerSrc } = prevTrailer;
-	const { src: nextTrailerSrc } = nextTrailer;
-
+  
 	return (
-		<aside>
-			<a
-				href={ trailerSrc }
-				aria-controls={ previewId }
-				aria-label={ `Watch preview for episode ${episodeNo}: ${title}.` }
-				onClick={ openOverlay }
-			>
-				<img
-					src={ thumbSrc }
-					alt={ thumbDesc }
-				/>
-			</a>
-			<div
-				id={ previewId }
-				className= { s.wrapper }
-				aria-hidden={ hidden }
-			>
-				<header>
-					<h2>
-						{ `Episode ${episodeNo}` } 
-					</h2>
-					<h1>
-						{ title }
-					</h1>
-					<nav>
-						{ !isFirst && (
-							<a 
-								href={ prevTrailerSrc }
-								aria-label="Previous."
-								onClick={ prevEpisode }
-							>
-								Prev
-							</a>
-						)}
-						{ !isLast && (
-							<a 
-								href={ nextTrailerSrc } 
-								onClick={ nextEpisode }
-							>
-								Next
-							</a>
-						)}
-						<a 
-							href={ `#${episodeId}` }
-							onClick={ closeOverlay }
-						> 
-							Close
-						</a>
-					</nav>
-				</header>
-				<video>
-					{ isOverlayActive && (
-						<source 
-							src={trailerSrc}
-							type="video/mp4"
+		<aside className={ s.wrapper }>
+			<div className={ s.thumbnail }>
+				<a
+					className= { s.button }
+					href={ trailerSrc }
+					aria-controls={ previewId }
+					aria-label={ `Watch preview for episode ${episodeNo}: ${title}.` }
+					onClick={ openOverlay }
+				>
+					<img
+						className={ s.image }
+						src={ thumbSrc }
+						alt={ thumbDesc }
+					/>
+					<div className={ s.icon }>
+						<PlayIcon 
+							colour="#FFF" 
 						/>
```

Comment:
Ah, almost - this isn't quite what I meant- howcome we should just be able to do 
```
<PlayIcon
    className={s.icon}
    colour="'#FFF"
/>
```

Reply:
that doesn't actually do anything, unless we unpack the classname in the Play component and then use that as the classname. Which is what we had before.

The classname for the SVG is `icons icon`, giving the PlayIcon a className just passed that as a prop, it doesn't actually allow us to select it in our PreviewOverlay styling.